sdcm/rest/storage_service_client.py

```diff
@@ -0,0 +1,39 @@
+from typing import Optional
+
+from fabric.runners import Result
+
+from sdcm.cluster import BaseNode
+from sdcm.rest.remote_curl_client import RemoteCurlClient
+
+
+class StorageServiceClient(RemoteCurlClient):
+    def __init__(self, node: BaseNode):
+        super().__init__(host="localhost:10000", endpoint="storage_service", node=node)
+
+    def compact_ks_cf(self, keyspace: str, cf: str) -> Result:
+        params = {"cf": cf} if cf else {}
```

Comment:
Do we expect empty "cf" here? It's mandatory parameter

Reply:
Nope. Fixed - made it optional and default to `None`.